pkg/cmd/notebook/notebook.go

```diff
@@ -54,16 +54,15 @@ func NewCmdNotebook(store NotebookStore, _ *terminal.Terminal) *cobra.Command {
 
 			// Check if the workspace is running
 			if result.Workspace.Status != "RUNNING" {
-				hello.TypeItToMeUnskippable27("The workspace is not running. Please ensure it's in the running state before proceeding.")
 				return breverrors.WorkspaceNotRunning{Status: result.Workspace.Status}
 			}
 
 			urlType := color.New(color.FgCyan, color.Bold).SprintFunc()
 			warningType := color.New(color.FgBlack, color.Bold, color.BgCyan).SprintFunc()
 
-			hello.TypeItToMeUnskippable("\n" + warningType("  Please keep this terminal open 🤙  "))
+			fmt.Print("\n" + warningType("  Please keep this terminal open 🤙  "))
 
-			hello.TypeItToMeUnskippable27("\nClick here to go to your Jupyter notebook:\n\t 👉" + urlType("http://localhost:8888") + "👈\n\n\n")
+			fmt.Print("\nClick here to go to your Jupyter notebook:\n\t 👉" + urlType("http://localhost:8888") + "👈\n\n\n")
```

Comment:
$0.02: We can drop the "here" part of click here. People know they can click the url. 
```suggestion
			fmt.Print("\nClick to go to your Jupyter notebook:\n\t 👉" + urlType("http://localhost:8888") + "👈\n\n\n")
```